snapcraft/plugins/copy.py

```diff
@@ -65,6 +67,13 @@ def schema(cls):
 
         return schema
 
+    def __init__(self, name, options, project):
+        super().__init__(name, options, project)
+
+        logger.warning("DEPRECATED: The 'copy' plugin's functionality "
+                       "has been replaced by the 'dump' plugin, and it will "
```

Comment:
El 05/08/16 a las 16:50, Kyle Fazzari escribió:
> In snapcraft/plugins/copy.py
> <https://github.com/snapcore/snapcraft/pull/712#discussion_r73749912>:
> 
>> @@ -65,6 +67,13 @@ def schema(cls):
>>  
>>          return schema
>>  
>> +    def __init__(self, name, options, project):
>> +        super().__init__(name, options, project)
>> +
>> +        logger.warning("DEPRECATED: The 'copy' plugin's functionality "
>> +                       "has been replaced by the 'dump' plugin, and it will "
> 
> I think that's too much info for a deprecation notice. If they want to
> learn more they can look at the docs for that plugin.

It is the same text used for tar-content